responder/api.py

```diff
@@ -54,6 +55,7 @@ def __init__(
         cors=False,
         cors_params=DEFAULT_CORS_PARAMS,
         allowed_hosts=None,
+        api_theme=DEFAULT_OPENAPI_THEME,
```

Comment:
Its still here, everywhere api_theme is used should be replaced with `openapi_theme` and same thing for `DEFAULT_API_THEME` to `DEFAULT_OPENAPI_THEME`

Reply:
Right, another spot has now been fixed, thanks for spotting. 80eebd5b1d0 includes all the renames.